src/components/ErrorNotification.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from 'classnames';
+import { useEffect, useState } from 'react';
+
+type Props = {
+  hasLoadingError: boolean,
+  setHasLoadingError: (arg: boolean) => void,
+  isAddingErrorShown: boolean,
+  setIsAddingErrorShown: React.Dispatch<React.SetStateAction<boolean>>,
```

Comment:
Write code in one style. If you decide to type setState actions with Dispatch<SetStateAction<type>> - do it everywhere. It'll be easier to understand that's just setState action from the first sight.